nextcord/invite.py

```diff
@@ -363,6 +369,8 @@ def __init__(
             PartialAppInfo(data=application, state=state) if application else None
         )
 
+        self.flags: Optional[InviteFlagsPayload] = data.get("flags")
```

Comment:
You sure this is the right type to be exposing? It should also simply default to flag value `0`. Might also be good for some helper functions like `Role.is_bot_managed`.